guides/v2.1/cloud/project/project-conf-files_magento-app.md

```diff
@@ -79,18 +79,27 @@ The `web` property defines how your application is exposed to the web (in HTTP).
 
 You can specify the following attributes for the `web` property:
 
--  `document_root`: The path relative to the root of the application that is exposed on the web. Typical values include `/public` and `/web`.
--  `passthru`: The URL used in the event a static file or PHP file could not be found. This would typically be your applications front controller, often `/index.php` or `/app.php`.
--  `index_files`: To use a static file (for example, `index.html`) to serve your application. This key expects a collection. For this to work, the static file(s) should be included in your whitelist. For example, to use a file named `index.html` as an index file, your whitelist should include an element that matches the filename, like `- \.html$`.
--  `blacklist`: A list of files that should never be executed. Has no effect on static files.
--  `whitelist`: A list of static files (as regular expressions) that can be served. Dynamic files (for example, PHP files) are treated as static files and have their source code served, but they are not executed.
--  `expires`: The number of seconds whitelisted (that is, static) content should be cached by the browser. This enables the cache-control and expires headers for static content. The `expires` directive and resulting headers are left out entirely if this is not set.
+Attribute | Description
+--------- | -----------
+`document_root` | The path relative to the root of the application that is exposed on the web. Typical values include `/public` and `/web`.
+`passthru` | The URL used in the event a static file or PHP file could not be found. This would typically be your applications front controller, often `/index.php` or `/app.php`.
```

Comment:
```suggestion
`passthru` | The URL used in the event that a static file or PHP file cannot be found. This URL is typically the front controller for your applications, often `/index.php` or `/app.php`.
```

Reply:
Question-- In our docs, I noticed that we use URL sometimes when we mean path --    
/index.php is a path.     In this case, I think that the path specified is appended to the site URL. Maybe this could be clearer. Maybe I am splitting hairs?